src/components/RegistrationFormBody.tsx

```diff
@@ -8,12 +8,12 @@ const RegistrationFormBody: React.FC<{
   field: FormListFieldData;
   isChildForm: boolean;
 }> = ({ field, isChildForm }) => {
-  const fieldWithoutKey: FormListFieldData = field;
-  delete fieldWithoutKey.key;
+  // const fieldWithoutKey: FormListFieldData = field;
+  // delete fieldWithoutKey.key;
   return (
     <>
       <Form.Item
-        {...fieldWithoutKey}
```

Comment:
@anirudh723 why did you delete this? Did you test to see if the form still works?

Reply:
The build was complaining because `fieldWithoutKey` was of `any` type, so when I changed it to have an actual type (`FormListFieldData`), it was complaining when deleting the `key` property because it was a required property. I looked at the html that was using this variable, and it looked like it was already extracting the `name` and `fieldKey` of the field anyway... so this destructuring this seemed unnecessary. 

Wasn't totally sure though, which is why I only commented it out

Reply:
if we're gonna leave it commented out, we should delete it